doc/dbus.rst

```diff
@@ -291,6 +291,14 @@ org.freedesktop.ratbag1.Resolution
 
         Index of the resolution
 
+.. attribute:: Capabilities
+
+        :type: au
+        :flags: read-only, constant
+
+	Array of uints from the :cpp:enum:`ratbag_resolution_capability`
+	from libratbag.h.
```

Comment:
Just noticed, this has tabs instead of spaces.

Reply:
Good spot! While fixing I found some more, see #964.